src/main/java/com/isn/quizplatform/model/Question.java

```diff
@@ -0,0 +1,69 @@
+package com.isn.quizplatform.model;
+
+import jakarta.persistence.*;
+import java.util.List;
+
+
+/**
+ * Classe question
+ */
+@Entity
+@Table(name="question")
+public class Question {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String libelle;
+
+    @ManyToOne(fetch = FetchType.LAZY) // Permet la relaiton avec la classe quiz en mode lazy
```

Comment:
La relation serait plutot many to many 
![image](https://github.com/user-attachments/assets/b775636e-eb39-4194-9064-39362fd98252)
Et je ne pense pas que ça serait intéressant d'avoir accès à ces données dans ce sens là (De question accéder au Quiz)